libcontainer/dmz/_dmz.c

```diff
@@ -11,5 +14,14 @@ int main(int argc, char **argv)
 {
 	if (argc < 1)
 		return 127;
-	return execve(argv[0], argv, environ);
+	int ret = execve(argv[0], argv, environ);
+	if (ret) {
+		char err_msg[5 + PATH_MAX] = "exec ";	// "exec " + argv[0]
+
+		strcpy(err_msg + strlen(err_msg), argv[0]);
```

Comment:
Needs to be `strlcpy`. `argv[0]` is attacker-controlled.

```suggestion
		strlcpy(err_msg + strlen(err_msg), argv[0], PATH_MAX);
```

Reply:
Ouch, good point. Updated, thanks!